test/integration-new/snat/snat_suite_test.go

```diff
@@ -0,0 +1,146 @@
+package snat
```

Comment:
IMO, this should be part of e2e test given we are crating and tearing down new nodes for the test.